contrib/kube-prometheus/examples/jsonnet-snippets/kops-coredns.jsonnet

```
@@ -0,0 +1,3 @@
+(import 'kube-prometheus/kube-prometheus.libsonnet') +
+(import 'kube-prometheus/kube-prometheus-kops.libsonnet') + 
+(import 'kube-prometheus/kube-prometheus-kops-coredns.libsonnet') 
```

```suggestion
(import 'kube-prometheus/kube-prometheus-kops-coredns.libsonnet')
```
You should be able to click "commit" here and that should resolve the CI failure.